aidants_connect_web/models.py

```diff
@@ -430,7 +463,8 @@ class Journal(models.Model):
     access_token = models.TextField(blank=True, null=True)
     mandat = models.IntegerField(blank=True, null=True)
     mandat_print_hash = models.CharField(max_length=100, blank=True, null=True)
-
+    additional_information = models.TextField(blank=True, null=True)
+    is_remote_mandat = models.BooleanField(default=False, blank=False, null=True)
```

Comment:
My bad, du coup, j'avais pas vu ce `null=True` lors de ma review.
Pour figer ici notre échange Slack, je pense qu'il est vraiment préférable d'éviter cela sur un `BooleanField`, à moins que tu veuilles explicitement autoriser un 3ème choix dans ta logique applicative. Et de fait, `blank=False` n'a pas vraiment de sens dans ce contexte, d'où cette suggestion :

```suggestion
    is_remote_mandat = models.BooleanField(default=False)
```